cmd/simulateEvol/simulateEvol.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/expandedTree"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/simulate"
+	"log"
+)
+
+func SimulateEvol(rootFastaFile string, treeFile string, simOutFile string, leafOutFile string) {
+	tree := expandedTree.ReadTree(treeFile, rootFastaFile)
+	var fastas []*fasta.Fasta
+	simulate.Simulate(rootFastaFile, tree)
```

Comment:
Not to do now, but I think it might make sense in the long-term to bring simulate and extendedTree into the reconstruct package since they really cater to that code.  Like I said, not now, longer term idea.